BUILD.md

```diff
@@ -66,39 +66,7 @@ If you enabled support for Pelion firmware updates in mbed-edge-core, copy your
 
 To generate update_default_resources.c, run [manifest-tool](https://github.com/ARMmbed/manifest-tool).  See the documentation on [getting the update resources](https://github.com/ARMmbed/mbed-edge/blob/master/README.md#getting-the-update-resources).
 
-
-#### Upgrade CA Certificate
-
-Authenticated upgrade requires inclusion of a cerificate authority certificate to be included in the initialization image.   This CA is used to issue the certificates included with an authenticated upgrade.  After obtaining your CA certificate, copy to the local file meta-pelion-edge/recipes-core/ww-console-image-initramfs-init/files/upgradeCA.cert.
-
-To generate upgradeCA.cert, create a CA key and use the key to generate a certificate signing request, or a self-signed certificate for development purposes.  Here is an example that shows how to create a key and a self-signed certificate.
-
-```
-openssl genrsa -out upgradeCA.key 2048
-openssl req -new -x509 -key upgradeCA.key -out upgradeCA.cert
-```
-
-
-#### Secure Boot Trusted World Root of Trust
-
-The Secure Boot RoT key is used to protect a subset of trusted world bootloaders as well as normal world U-Boot.
-
-To generate rot_key.pem, run openssl to generate an RSA key.
-
-```
-openssl genrsa -out rot_key.pem 2048
-```
-
-#### U-Boot Verified Boot FIT image signing key and certificate
-
-The FIT image signing key is used to protect U-Boot FIT images containing the kernel, DTBs, initramfs, and boot script.
-
-To generate mbl-fit-rot-key.key and mbl-fit-rot-key.crt, run the following openssl commands.
-
-```
-openssl genrsa -out mbl-fit-rot-key.key 2048
-openssl req -new -x509 -key mbl-fit-rot-key.key -out mbl-fit-rot-key.crt
-```
+Note: To unlock the rich node features, such as gateway logs and gateway terminal in the Pelion web portal please specify the `vendor-id=42fa7b48-1a65-43aa-890f-8c704daade54`
```

Comment:
I believe this will no longer be true very soon.  Do you know the timing of that?  Should we still say something here?

Reply:
Ya I think it will no longer be valid in about a month. I think we should keep this line until then.